adapter_nrf51.go

```diff
@@ -50,6 +50,19 @@ func handleEvent() {
 			currentConnection.Reg = gapEvent.conn_handle
 			DefaultAdapter.connectHandler(Address{}, true)
 		case C.BLE_GAP_EVT_DISCONNECTED:
+			if debug {
+				println("evt: disconnected")
+			}
+			// Store system attributes data
+			dataLen := uint16(0)
+			C.sd_ble_gatts_sys_attr_get(gapEvent.conn_handle, nil, &dataLen, 0) // get data length
+			if int(dataLen) <= cap(DefaultAdapter.systemAttributes) {           // we can not allocate here, so ensure at least data fits the buffer
```

Comment:
@aykevl I may need your help to solve this. We don't know in advance how large system attributes blob will be and we can't allocate here.
So far, I've seen sizes around 16 bytes, but I assume size depends on the number of characteristics published.
In this PR, I'm pre-allocating 64 bytes array and just skipping the storing logic if the blob is larger. I'm sure this can be handled better, but I don't know how.